modules/artifact/src/main/java/org/mule/module/artifact/classloader/JdbcResourceReleaser.java

```diff
@@ -48,7 +49,9 @@
  */
 public class JdbcResourceReleaser implements ResourceReleaser {
 
-  private static final String avoidThreadDisposal = getProperty("avoid.dispose.oracle.threads");
+  private static final String AVOID_DISPOSE_ORACLE_THREADS_PROPERTY_NAME = "avoid.dispose.oracle.threads";
+  private static final boolean JDBC_RESOURCE_RELEASER_AVOID_DISPOSE_ORACLE_THREADS =
```

Comment:
You should only use getBoolean without the getProperty

getBoolean(AVOID_DISPOSE_ORACLE_THREADS_PROPERTY_NAME)

Return true if and only if the system property is true.